server/AuthClientMultiCredentials.js

```diff
@@ -0,0 +1,69 @@
+const OAuth2 = require('../node_modules/forge-apis/src/auth/OAuth2');
```

Comment:
@nick-ooms-neanex Shouldn't `forge-apis` be a dependency (if it isn't already) instead, so you don't have to require a `node_modules` path?

Reply:
OAuth2 isn't exported by the forge-apis
https://github.com/Autodesk-Forge/forge-api-nodejs-client/blob/master/src/index.js#L1008